<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1123</width>
    <height>744</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="cart">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>981</width>
      <height>631</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab_4">
     <attribute name="title">
      <string>Sign In</string>
     </attribute>
     <widget class="QWidget" name="verticalLayoutWidget_3">
      <property name="geometry">
       <rect>
        <x>190</x>
        <y>90</y>
        <width>776</width>
        <height>451</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_10">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_8">
           <item>
            <widget class="QLabel" name="label">
             <property name="text">
              <string>Operator Username: </string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_9">
           <item>
            <widget class="QLineEdit" name="operatorIDLineEdit"/>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QGridLayout" name="gridLayout_5">
         <item row="0" column="1">
          <widget class="QLabel" name="professorMessageLabel">
           <property name="text">
            <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Hello Professor, the Operator ID can be whatever you want it to be. &lt;/p&gt;&lt;p&gt;The Operator ID will be present in the transaction receipt.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
           </property>
          </widget>
         </item>
         <item row="0" column="0">
          <widget class="QPushButton" name="cancelButton3">
           <property name="text">
            <string>Cancel</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Store</string>
     </attribute>
     <widget class="QWidget" name="gridLayoutWidget">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>951</width>
        <height>581</height>
       </rect>
      </property>
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="0">
        <widget class="QLabel" name="pinkJackLabel">
         <property name="text">
          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Pink Jacket &lt;/p&gt;&lt;p&gt;Price: $30&lt;/p&gt;&lt;p&gt;Item ID: 1&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QSpinBox" name="yJSpinBox"/>
       </item>
       <item row="4" column="0">
        <widget class="QLabel" name="pShoesLabel">
         <property name="text">
          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Pink Shoes&lt;/p&gt;&lt;p&gt;Price: $30&lt;/p&gt;&lt;p&gt;Item ID: 7&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
        </widget>
       </item>
       <item row="4" column="1">
        <widget class="QLabel" name="yShoesLabel">
         <property name="text">
          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Yellow Shoes&lt;/p&gt;&lt;p&gt;Price: $22&lt;/p&gt;&lt;p&gt;Item ID: 8&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
        </widget>
       </item>
       <item row="2" column="2">
        <widget class="QLabel" name="blueShirtLabel">
         <property name="text">
          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Blue Shirt&lt;/p&gt;&lt;p&gt;Price: $11&lt;/p&gt;&lt;p&gt;Item ID: 6&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
        </widget>
       </item>
       <item row="5" column="2">
        <widget class="QSpinBox" name="blueShoesSpinBox"/>
       </item>
       <item row="3" column="2">
        <widget class="QSpinBox" name="blueShirtSpinBox"/>
       </item>
       <item row="6" column="1">
        <widget class="QPushButton" name="addToCart">
         <property name="text">
          <string>Add To Cart</string>
         </property>
        </widget>
       </item>
       <item row="0" column="2">
        <widget class="QLabel" name="bueJacketLabel">
         <property name="text">
          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Blue Jacket&lt;/p&gt;&lt;p&gt;Price: $ 20&lt;/p&gt;&lt;p&gt;Item ID: 3&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QSpinBox" name="pShirtSpinBox"/>
       </item>
       <item row="1" column="2">
        <widget class="QSpinBox" name="bJSpinBox"/>
       </item>
       <item row="1" column="0">
        <widget class="QSpinBox" name="pJSpinBox"/>
       </item>
       <item row="5" column="1">
        <widget class="QSpinBox" name="yShoesSpinBox"/>
       </item>
       <item row="3" column="1">
        <widget class="QSpinBox" name="yShirtSpinBox"/>
       </item>
       <item row="5" column="0">
        <widget class="QSpinBox" name="pShoesSpinBox"/>
       </item>
       <item row="0" column="1">
        <widget class="QLabel" name="yellowJacketLabel">
         <property name="text">
          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Yellow Jacket&lt;/p&gt;&lt;p&gt;Price: $25&lt;/p&gt;&lt;p&gt;Item ID: 2&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
        </widget>
       </item>
       <item row="4" column="2">
        <widget class="QLabel" name="bShoesLabel">
         <property name="text">
          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Blue Shoes&lt;/p&gt;&lt;p&gt;Price: $ 40&lt;/p&gt;&lt;p&gt;Item ID: 9&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QLabel" name="yellowShirtLabel">
         <property name="text">
          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Yellow Shirt&lt;/p&gt;&lt;p&gt;Price: $12&lt;/p&gt;&lt;p&gt;Item ID: 5&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="pinkShirtLabel">
         <property name="text">
          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Pink Shirt&lt;/p&gt;&lt;p&gt;Price: $15&lt;/p&gt;&lt;p&gt;Item ID: 4&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>Cart</string>
     </attribute>
     <widget class="QWidget" name="formLayoutWidget">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>801</width>
        <height>511</height>
       </rect>
      </property>
      <layout class="QFormLayout" name="formLayout">
       <item row="0" column="1">
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <layout class="QVBoxLayout" name="verticalLayout">
           <item>
            <widget class="QLabel" name="itemLabel">
             <property name="text">
              <string>Items:</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_2">
           <item>
            <widget class="QLabel" name="itemListLabel">
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item row="1" column="1">
        <layout class="QGridLayout" name="gridLayout_2">
         <item row="0" column="1">
          <widget class="QLabel" name="totalPrice">
           <property name="text">
            <string>Total Price: </string>
           </property>
          </widget>
         </item>
         <item row="0" column="0">
          <widget class="QPushButton" name="clearButton">
           <property name="text">
            <string>Clear</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QPushButton" name="checkoutButton">
           <property name="text">
            <string>Checkout</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QPushButton" name="cancelButton">
           <property name="text">
            <string>Cancel</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_3">
     <attribute name="title">
      <string>Checkout</string>
     </attribute>
     <widget class="QWidget" name="formLayoutWidget_2">
      <property name="geometry">
       <rect>
        <x>230</x>
        <y>150</y>
        <width>391</width>
        <height>401</height>
       </rect>
      </property>
      <layout class="QFormLayout" name="formLayout_3">
       <item row="0" column="1">
        <layout class="QHBoxLayout" name="horizontalLayout_7">
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_5">
           <item>
            <widget class="QLabel" name="nameLabel">
             <property name="text">
              <string>Name: </string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="AddressLabel">
             <property name="text">
              <string>Address: </string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="phoneNumberLabel">
             <property name="text">
              <string>Phone Number: </string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="emailLabel">
             <property name="text">
              <string>Email Address: </string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_6">
           <item>
            <widget class="QLineEdit" name="nameLineEdit">
             <property name="inputMask">
              <string>aaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaa</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="addressLineEdit">
             <property name="inputMask">
              <string/>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="phoneNumberLineEdit">
             <property name="inputMask">
              <string>9999999999</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="emailLineEdit">
             <property name="inputMask">
              <string>xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item row="1" column="1">
        <layout class="QGridLayout" name="gridLayout_3">
         <item row="0" column="1">
          <widget class="QPushButton" name="clearButton2">
           <property name="text">
            <string>Clear</string>
           </property>
          </widget>
         </item>
         <item row="0" column="0">
          <widget class="QPushButton" name="cancelButton2">
           <property name="text">
            <string>Cancel</string>
           </property>
          </widget>
         </item>
         <item row="0" column="2">
          <widget class="QPushButton" name="submitButton">
           <property name="text">
            <string>Submit</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1123</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
